internal/runner/runner.go

```diff
@@ -156,6 +156,13 @@ func New(opts ...Option) (*Runner, error) {
 	return runner, nil
 }
 
+func (r *Runner) LoadLocalProjectPlugins(path string) error {
+	// TODO: should this check if the runner is local?
+	err := r.plugins.LoadLocalLegacyPlugins(
```

Comment:
I'm not too sure about this approach. If the runner is local then it should be able to load local project plugins. Not sure how to resolve the situation where the runner is remote?